app/configurator/components/chart-options-selector.tsx

```diff
@@ -120,13 +150,15 @@ const EncodingOptionsPanel = ({
   chartType,
   component,
   metaData,
+  imputationNeeded,
 }: {
   encoding: EncodingSpec;
   state: ConfiguratorStateConfiguringChart;
   field: string;
   chartType: ChartType;
   component: DimensionMetaDataFragment | undefined;
   metaData: DataCubeMetadata;
+  imputationNeeded: boolean;
```

Comment:
Not a fan of this though (prop drilling the imputationNeeded), it seems like it should be part of a more generic context that would contain computed informations on the chart. The generic "stats" computation context would hold imputationNeeded, and then component that need it would just take it from the context.

Let's keep it in mind when we do further improvements, I do not think it is needed at this point.